src/components/App/Main/LevelUpCode/LevelUpCode.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import S from './LevelUpCode.module.css';
+
+function LevelUpCode(){
+	return(
+		<section className={S.level_up_code_box}>
+			<div className={S.level_up_code_content}>
+				<div className={S.code_example}>
+					<div className={S.code_content}>
+						<pre className={S.code_content_first_line}>@SpringBootApplication  </pre>
+						<pre className={S.code_content_second_line}>@RestController  </pre>
+						<pre><span className={S.code_content_third_line}>public</span> class DemoApplication {"{"}  </pre>
+						<pre className={S.code_content_fourth_line}>   <span className={S.code_content_fourth_line_part_1}>@GetMapping</span><span className={S.code_content_fourth_line_part_2}>("/helloworld")</span>  </pre>
+						<pre>   <span className={S.code_content_fifth_line}>public</span> String hello() {"{"}  </pre>
+						<pre>      return <span className={S.code_content_sixth_line}>"Hello World!"</span>;  </pre>
+						<pre>   {"{"}  </pre>
+						<pre>{"{"}  </pre>
+					</div>
```

Comment:
Pure awfulness, again. if you had 1000000000 code lines, you would create as many styles? Don't be ridiculous. You even had a nicely written example itself, not an image or a design mock-up.